pro/pro-users.rst

```diff
@@ -1,194 +1,82 @@
+* Super Admin: Super admins have full access to Netmaker. They can create and manage users, admins, networks, and gateways. They can also manage user and admin permissions.
 
-Level 2: Remote Access
-------------------------
-
-Lets say we add this user to a network, "mynet", with Level 2 access, and a limit of 2 ext clients. They will have Remote Access, and will be able to create and manage up to 2 ext clients.
+Adding users
+=================================
+To add a user, go to the Users section and click the Add User button. Fill in the user's details, including their name, password, and role.
 
-Logging in, they will see this screen:
+* As a super admin, you can add users with the role of admin or user.
+* As an admin, you can only add users with the role of user.
 
-.. image:: images/users/non-admin-dashboard-1.png
+.. image:: images/users/add-user.png
    :width: 80%
-   :alt: Welcome Screen
+   :alt: Add User
    :align: center
 
-Clicking continue, they will be prompted to select a network:
-
-.. image:: images/users/user-dashboard-1.png
-   :width: 80%
-   :alt: Select Network
-   :align: center
+The credentials will need to be shared with the added user.
 
-They will then be at their Dashboard, and will need to select an option. Since they are Level 2, they have only one option, "Remote Access:"
+Attaching or removing user from a client gateway
+================================================
+To attach users to a client gateway or remove users from a gateway, you will need to have the client gateway set up. 
+Once the client gateway is set up, you will see an option to remove users from the dropdown menu.
 
-.. image:: images/users/user-dashboard-2.png
+.. image:: images/users/gateway-dropdown.png
    :width: 80%
-   :alt: Remote Access 1
+   :alt: Add Remove User dropdown
    :align: center
 
-Clicking this brings them to the familiar "ext clients" screen. Here, they can create, manage, delete, view, and download their own ext clients for any Ingress Gateways on the network. At the top, it shows how many clients they have used, as opposed to their limit.
+You can click the button to either attach or remove a user
 
-.. image:: images/users/user-dashboard-3.png
+.. image:: images/users/attach-remove-users.png
    :width: 80%
-   :alt: Remote Access 2
+   :alt: Attach Remove User Modal
    :align: center
 
-After creating 2 clients, the user will be unable to create any more (in this example, their limit was set to "2").
-
 
-.. image:: images/users/user-dashboard-4.png
-   :width: 80%
-   :alt: Remote Access 3
-   :align: center
+Transferring super admin rights
+===============================
 
+Super admin rights can be transferred only to another admin. To do this, on the users page, go to the superadmin row and hover over the ellipsis.
+In the dropdown menu, an option to transfer admin rights should be present. On clicking it, a dialog box will open allowing you to select any admin 
```

Comment:
`In the dropdown menu, an option to transfer admin rights should be present` -> you will see an option to transfer admin rights`